traits/observation/_has_traits_helpers.py

```diff
@@ -129,8 +129,7 @@ def ctrait_prevent_event(event):
     """
     if event.old is Uninitialized:
         return True
-
-    ctrait = event.object.traits()[event.name]
+    ctrait = event.object.trait(event.name)
```

Comment:
Just as a highly personal nitpicky matter of style, I do like the blank line following the early return.

Reply:
It was accidentally, but then I was lazy to revert, sorry about that... reverting.